### PR TITLE
fixes #5969 - deleting of EC2 guests - confusing message

### PR DESCRIPTION
**Pull request by isratrade**

### CODE REVIEW

**Comment by isratrade** [comment]
@domcleal, is this destroy message true for all types of compute resources, or just EC2?

**Comment by domcleal** [comment]
@isratrade I don't know, please test.

**Comment by dmitri-d** [comment]
For the sake of avoiding per-VM provider deletion messages, may I suggest using different phrasing, something like: "It may take a few minutes to take effect".

**Comment by isratrade** [comment]
@witlessbird, how is the message 'The virtual machine will be deleted in a few minutes' connected with per-VM provider. Also note that the message is on the `destroy` action.

File: app/controllers/compute_resources_vms_controller.rb [hunk 1]
```diff
@@ -70,7 +70,7 @@ def pause
   def destroy
     @compute_resource = find_compute_resource(:destroy_compute_resources_vms)
     if @compute_resource.destroy_vm params[:id]
-      process_success({ :success_redirect => compute_resource_vms_path(@compute_resource) })
+      process_success({ :success_redirect => compute_resource_vms_path(@compute_resource), :success_msg => _('The virtual machine will be deleted in a few minutes') })
```
Comment (isratrade) [inline app/controllers/compute_resources_vms_controller.rb]: if the deletion only takes seconds (or less) for libvirt, then this message "in a few minutes" may be confusing.
Should we change it to
"The virtual machine is being deleted in the background"

Comment (domcleal) [inline app/controllers/compute_resources_vms_controller.rb]: Yeah, or simply "The virtual machine is being deleted."

**Comment by isratrade** [comment]
re-committed with message "The virtual machine is being deleted."

**Comment by GregSutcliffe** [comment]
Tested locally, works as described. +1 to merge

**Comment by domcleal** [comment]
Merged as 9d9523c for Foreman 1.6.0, thanks @isratrade.

**Pull request closed without merging**